resources/event/event.controller.js

```diff
@@ -0,0 +1,29 @@
+const Event = require("./event.model");
+const ScrapeEvents = require("../../utils/webScraper");
+
+const addScrapedEvent = async () => {
+  const scrappedEventsArray = await ScrapeEvents;
```

Comment:
Could you elaborate on this line, I think `ScrapeEvents` it should be invoked as a function.

Reply:
This is a Promise. We tried invoking earlier but it did not work.
We will be happy to get some insight on this.